code/liam/html/lab05/app.py

```diff
@@ -0,0 +1,88 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+ones_table = {
+    1: 'one',
+    2: 'two',
+    3: 'three',
+    4: 'four',
+    5: 'five',
+    6: 'six',
+    7: 'seven',
+    8: 'eight',
+    9: 'nine',
+    0: '',
+    10: 'ten',
+    11: 'eleven',
+    12: 'twelve',
+}
+
+tens_table = {
+    0: 'oh',
+    1: 'null',
+    2: 'twenty',
+    3: 'thirty',
+    4: 'forty',
+    5: 'fifty',
+    6: 'sixty',
+    7: 'seventy',
+    8: 'eighty',
+    9: 'ninety',
+}
+
+tenteens_table = {
+    10: 'ten',
+    11: 'eleven',
+    12: 'twelve',
+    13: 'thirteen',
+    14: 'fourteen',
+    15: 'fifteen',
+    16: 'sixteen',
+    17: 'seventeen',
+    18: 'eighteen',
+    19: 'nineteen',
+}
+
+
+@app.route('/')
+def num_to_phrase():
+    return render_template('index.html')
+
+@app.route('/phrase/', methods=['GET', 'POST'])
```

Comment:
I don't think you actually want to be able to make GET requests to this endpoint